config/html-webpack-inline-critical-css-plugin.js

```diff
@@ -0,0 +1,255 @@
+const fs = require('fs');
+const path = require('path');
+const parse5 = require('parse5');
+const nwmatcher = require('nwmatcher');
+const css = require('css');
+
+const treeUtils = parse5.treeAdapters.htmlparser2;
+
+const PLUGIN_NAME = 'html-webpack-inline-critical-css-plugin';
+
+const PARSE5_OPTS = {
+  treeAdapter: treeUtils
+};
+
+function defineProperties(obj, properties) {
```

Comment:
Why are you polyfilling `defineProperties`?

Reply:
Wasn't - most of the values are functions that get assigned as pure `value`s, this made that a bit easier (rather than having to specific `{ value() {} }` for each prototype method.

Reply:
Oh I see. Comment please :D